azure/table_azure_databricks_workspace.go

```diff
@@ -0,0 +1,224 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/databricks/mgmt/2018-04-01/databricks"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureDatabricksWorkspace(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_databricks_workspace",
+		Description: "Azure Databricks Workspace",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "resource_group"}),
+			Hydrate:    getDatabricksWorkspace,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"ResourceNotFound", "ResourceGroupNotFound", "400"}),
+			},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listDatabricksWorkspaces,
+		},
+		Columns: azureColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The resource name.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "Fully qualified resource Id for the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromGo(),
+			},
+			{
+				Name:        "sku",
+				Description: "The SKU of the resource.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "type",
+				Description: "The type of the resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "location",
+				Description: "The geo-location where the resource lives.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "managed_resource_group_id",
+				Description: "The managed resource group Id.",
```

Comment:
```suggestion
				Description: "The managed resource group ID.",
```
Could you please update all instances of `Id` to `ID`?